include/zephyr/usb/class/usb_cdc.h

```diff
@@ -85,6 +85,22 @@
 #define SERIAL_STATE_TX_CARRIER		0x02
 #define SERIAL_STATE_RX_CARRIER		0x01
 
+/** Line Coding Structure Values */
```

Comment:
Maybe mention here where they come from.

```
/**
 * @brief PSTN Subclass Line Coding Values
 * 
 * @note PSTN120.pdf, 6.3.11, Table 17
 */
```

Reply:
Thanks for the review, I have just done the suggested changes.